_includes/releases/v21.2/v21.2.8.md

```diff
@@ -0,0 +1,101 @@
+- Fixed a bug where a node in the `UNAVAILABLE` state would not have latency defined and caused the network page to crash. [#78627][#78627]
+
+<h3 id="v21-2-8-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that caused errors when attempting to create table statistics (with [`CREATE STATISTICS`](../v21.2/create-statistics.html) or [`ANALYZE`](../v21.2/explain-analyze.html)) for a table containing an index that indexed only virtual [`computed columns`](../v21.2/computed-columns.html). This bug has been present since version 21.1.0. [#77565][#77565]
+- Fixed a bug when adding a [`hash-sharded index`](../v21.2/hash-sharded-indexes.html) to a table watched by a changefeed. [#77739][#77739]
+- Fixed successive schema change backfills from skipping spans that were checkpointed by an initial backfill that was restarted. [#77829][#77829]
+- Attempting to run concurrent profiles now works up to a concurrency limit of two. This will remove the occurrence of `profile id not found` errors while running up to two profiles concurrently. When a profile is not found, the error message has been updated to suggest troubleshooting steps. [#77977][#77977]
+- Fixed an optimizer bug that prevented expressions of the form `(NULL::STRING[] <@ ARRAY['x'])` from being folded to `NULL`. This bug was introduced in v21.2.0. [#78039][#78039]
+- Added a limit of seven concurrent asynchronous consistency checks per store, with an upper timeout of one hour. This prevents abandoned consistency checks from building up in some circumstances, which could lead to increasing disk usage as they held onto Pebble snapshots. [#77611][#77611]
+- Fixed a bug where the [**Statement Details**](../v21.2/ui-statements-page.html#statement-details-page) page fails to load query plan even after when the plan has been sampled. [#78105][#78105]
+- Fixed a memory leak in the Pebble block cache. [#78257][#78257]
+- Fixed a bug that caused internal errors when `COALESCE` and `IF` expressions had inner expressions with different types that could not be cast to a common type. [#78342][#78342]
+- CockroachDB might now fetch fewer rows when performing lookup and index joins on queries with the [`LIMIT`](../v21.2/limit-offset.html#limit) clause. [#78474][#78474]
+- A zone config change event now includes the correct details of what was changed instead of incorrectly displaying "undefined". [#78634][#78634]
+- Fixed a bug that prevented a table created on a 22.1 node to be queried on a 21.2 node in a mixed-version cluster. [#78657][#78657]
+- Fixed a bug that caused errors when trying to evaluate queries with `NULL` values annotated as a tuple type, such as `NULL:::RECORD`. This bug was present since version 19.1. [#78635][#78635]
```

Comment:
```suggestion
- Fixed a bug that caused errors when trying to evaluate queries with `NULL` values annotated as a tuple type, such as `NULL:::RECORD`. This bug was present since v19.1. [#78635][#78635]
```